build.xml

```diff
@@ -60,8 +65,8 @@
     </target>
 
 
-    <!-- Target: deploy-to-acquia -->
-    <target name="deploy-to-acquia">
+    <!-- Target: deploy-to-pantheon -->
+    <target name="deploy-to-pantheon">
```

Comment:
This whole target can go away, since you've put it in the `deploy` target above.